spec/requests/api/v1/docs/followed_tags_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+require "swagger_helper"
+
+# rubocop:disable RSpec/EmptyExampleGroup
+# rubocop:disable RSpec/VariableName
+
+RSpec.describe "Api::V1::Docs::FollowedTags" do
+  let(:Accept) { "application/vnd.forem.api-v1+json" }
+  let(:api_secret) { create(:api_secret) }
+  let(:user) { api_secret.user }
+  let(:tag1) { create(:tag) }
+  let(:tag2) { create(:tag) }
+
+  before {
+    user.follow(tag1)
+    user.follow(tag2)
+  }
+
+
+  describe "GET /follows/tags" do
+    path "/api/follows/tags" do
+      get "Followed Tags" do
+        security []
+        tags "followed_tags"
+        description(<<-DESCRIBE.strip)
+        This endpoint allows the client to retrieve a list of the tags they follow.
+        DESCRIBE
+        operationId "getFollowedTags"
+        produces "application/json"
+
+        response "200", "A List of followed tags" do
+          let(:"api-key") { nil }
+          schema  type: :array,
+                  items: { "$ref": "#/components/schemas/FollowedTags" }
```

Comment:
@lboogie04 where is this schema from? Should FollowedTags be defined in the swagger_helper?